### PR TITLE
#634: Missing payment method Error

### PR DESCRIPTION
**Pull request by hani-ibrahim**
fixing #634

### CODE REVIEW

**Comment by hani-ibrahim** [comment]
👍

File: AtlasUI/AtlasUI/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -8,7 +8,7 @@
 "AtlasCheckoutError.message.addressInvalid" = "Address not valid, please check the address";
 "AtlasCheckoutError.message.checkoutFailure" = "An error occurred during updating the checkout";
 "AtlasCheckoutError.message.missingAddress" = "Please select shipping and billing addresses first";
-"AtlasCheckoutError.message.missingAddressAndPayment" = "Please select shipping and billing addresses and the payment method first";
+"AtlasCheckoutError.message.missingPaymentMethod" = "Please the payment method first";
```
Comment (bonkey) [inline AtlasUI/AtlasUI/en.lproj/Localizable.strings]: Don't forget to update `translations` branch and remove old translations

**Comment by bonkey** [comment]
👍

**Comment by hani-ibrahim** [comment]
👍

**Pull request merged**